src/clj/game/cards/ice.clj

```diff
@@ -2060,6 +2060,16 @@
                                     :msg (msg "force the Runner to lose " (:tag runner) " [Credits]")
                                     :effect (effect (lose-credits :runner (:tag runner)))})]}
 
+   "Thimblerig"
+   {:flags {:corp-phase-12 (req (>= (count (filter ice? (all-installed state :corp))) 2))}
+    :implementation "Does not restrict usage of swap ability to start of turn or after pass"
+    :abilities [{:label "Swap Thimblerig with a piece of ice"
+                 :prompt "Swap Thimblerig with a piece of ice"
```

Comment:
Maybe have this say "Choose a piece of ice to swap Thimblerig with?"

Reply:
Makes sense. I'll fix it soonish.